stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/viewmodel/ChannelListViewModel.kt

```diff
@@ -59,10 +60,23 @@ public class ChannelListViewModel(
                                 isLoading = false,
                                 channels = emptyList(),
                             )
-                            is QueryChannelsController.ChannelsState.Result -> currentState.copy(
-                                isLoading = false,
-                                channels = channelState.channels.filterNot { it.hidden == true || it.isDraft },
-                            )
+                            is QueryChannelsController.ChannelsState.Result -> {
+                                val channelMutesIds = chatDomain.currentUser.channelMutes.map { it.channel.id }
+
+                                val channels = channelState
```

Comment:
What's about moving this logic in `QueryChannelsController`? It's more flexible to have such flag in domain logic instead only in one small UI part. The same code could be there when we update channels state flow